Arduino-Code/Arduino.md

```diff
@@ -0,0 +1,5 @@
+This is the folder to contain the codes that we run while ooperating on the machine during Srishti 2019.
+A brief overview-
```

Comment:
1. Rename this file as Readme.md
2. Description of each file should be written in points(numbered list).